CP05MOAS-GL340/CP05MOAS-GL340_R00009_ingest.csv

```diff
@@ -1,7 +1,7 @@
-parser,filename_mask,reference_designator,data_source,status,notes
-mi.dataset.driver.moas.gl.engineering.glider_eng_glider_recovered_driver,/omc_data/whoi/OMC/CP05MOAS-GL340/R00009/merged/cp*.mrg,CP05MOAS-GL340-00-ENG000000,recovered_host,Available,
-mi.dataset.driver.moas.gl.adcpa.adcpa_m_glider_recovered_driver,/omc_data/whoi/OMC/CP05MOAS-GL340/R00009/dvl/*.PD0,CP05MOAS-GL340-01-ADCPAM000,recovered_host,Available,
-mi.dataset.driver.moas.gl.flort_m.flort_m_glider_recovered_driver,/omc_data/whoi/OMC/CP05MOAS-GL340/R00009/merged/cp*.mrg,CP05MOAS-GL340-02-FLORTM000,recovered_host,Available,
-mi.dataset.driver.moas.gl.ctdgv.ctdgv_m_glider_recovered_driver,/omc_data/whoi/OMC/CP05MOAS-GL340/R00009/merged/cp*.mrg,CP05MOAS-GL340-03-CTDGVM000,recovered_host,Available,
-mi.dataset.driver.moas.gl.dosta.dosta_abcdjm_glider_recovered_driver,/omc_data/whoi/OMC/CP05MOAS-GL340/R00009/merged/cp*.mrg,CP05MOAS-GL340-04-DOSTAM000,recovered_host,Available,
+parser,filename_mask,reference_designator,data_source,status,notes
+mi.dataset.driver.moas.gl.engineering.glider_eng_glider_recovered_driver,/omc_data/whoi/OMC/CP05MOAS-GL340/R00009/merged/cp*.mrg,CP05MOAS-GL340-00-ENG000000,recovered_host,Available,
+#mi.dataset.driver.moas.gl.adcpa.adcpa_m_glider_recovered_driver,/omc_data/whoi/OMC/CP05MOAS-GL340/R00009/dvl/*.PD0,CP05MOAS-GL340-01-ADCPAM000,recovered_host,Not Available,DVL disabled; no data available

```

Comment:
No DVL data available for this deployment. commented out line, changed status and added comment